### PR TITLE
fix(lint):fix errcheck in admission-server

### PR DESCRIPTION
**Pull request by shubham14bajpai**
- handled `error` returned by `Shutdown` func

Signed-off-by: shubham <shubham.bajpai@mayadata.io>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
- [ ] Fixes #<issue number>
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `breaking-changes` tag
- [ ] PR messages has breaking changes related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] PR messages has upgrade related information
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: cmd/admission-server/main.go [hunk 1]
```diff
@@ -80,7 +80,9 @@ func main() {
 	<-signalChan
 
 	glog.Infof("Got OS shutdown signal, shutting down webhook server gracefully...")
-	wh.Server.Shutdown(context.Background())
+	if err := wh.Server.Shutdown(context.Background()); err != nil {
```
Comment (sonasingh46) [inline cmd/admission-server/main.go]: This way of error handling is not idiomatic.
```
err:=someFunc()
if err!=nil{
...
}
```

Comment (shubham14bajpai) [reply cmd/admission-server/main.go]: changes done sir.

File: cmd/admission-server/main.go [hunk 2]
```diff
@@ -80,7 +80,10 @@ func main() {
 	<-signalChan
 
 	glog.Infof("Got OS shutdown signal, shutting down webhook server gracefully...")
-	wh.Server.Shutdown(context.Background())
+	err = wh.Server.Shutdown(context.Background())
+	if err != nil {
+		glog.Errorf("Failed to shutdown server: %v", err)
```
Comment (AmitKumarDas) [inline cmd/admission-server/main.go]: can we rephrase as follows:
```go
glog.Error("failed to shutdown server: error {%v}", err)
```

Comment (shubham14bajpai) [reply cmd/admission-server/main.go]: Done sir.

**Pull request merged**